app/javascript/components/pages/key-performance-indicators/container.jsx

```diff
@@ -0,0 +1,83 @@
+import { fromJS } from "immutable";
+import PropTypes from "prop-types";
+import React, { useEffect } from "react";
+import { withRouter } from "react-router-dom";
+import { connect, batch } from "react-redux";
+import { PageContainer } from "components/page-container";
+import { Grid, Box } from "@material-ui/core";
+import { OptionsBox, DashboardTable } from "components/dashboard";
+import { useI18n } from "components/i18n";
+import makeStyles from "@material-ui/styles/makeStyles";
+import styles from "./styles.css";
+import * as actions from "./action-creators";
+import * as selectors from "./selectors";
+
+function KeyPerformanceIndicators({ fetchNumberOfCases, numberOfCases }) {
+  const i18n = useI18n();
+  const css = makeStyles(styles)();
+  useEffect(() => {
+    batch(() => {
+      fetchNumberOfCases();
+    });
+  }, []);
+
+  return (
+    <div>
+      <PageContainer>
+        <Grid item xs={12}>
+          <Box alignItems="center" display="flex">
+            <Box flexgrow={1}>
+              <h1 className={css.title}>
+                {i18n.t("key_performance_indicators.label")}
+              </h1>
+            </Box>
+          </Box>
+        </Grid>
+
+        <Grid>
+          <Box>
+            <h2 className={css.subtitle}>INTRODUCTION &amp; ENGAGEMENT</h2>
```

Comment:
Will this title come from the backend?

Reply:
I expect this to come from translations at the very least. It depends on how much of the reports infrastructure we end up using for KPIs though. I'm holding off on making that decision.